psga/common/common.config

```diff
@@ -27,7 +27,7 @@ env {
     QUEUE_SIZE = "${QUEUE_SIZE}"
     PROCESS_MAX_RETRIES = "${PROCESS_MAX_RETRIES?:'3'}"
     PROCESS_CPU_LOW = "${PROCESS_CPU_LOW?:'1'}"
-    PROCESS_CPU_HIGH = "${PROCESS_CPU_HIGH?:'2'}"
+    PROCESS_CPU_HIGH = "${PROCESS_CPU_HIGH?:'1'}"
```

Comment:
I think we should trigger a batch of 200 samples to determine the impact of this.

Reply:
Removed those changes, we can try them separately in isolation.